dto/src/main/java/org/jboss/pnc/dto/notification/BuildConfigurationCreationError.java

```diff
@@ -46,4 +42,11 @@ public BuildConfigurationCreationError(Integer repositoryId, Integer buildConfig
 
     @Getter
     private final String errorMessage;
+
+    public BuildConfigurationCreationError(Integer repositoryId, Integer buildConfigurationId, String errorMessage) {
+        super(BUILD_CONFIG_CREATION, BC_CREATION_ERROR, FINISHED);
+        this.repositoryId = repositoryId;
+        this.buildConfigurationId = buildConfigurationId;
```

Comment:
repositoryId should be scmRepositoryId
buildConfigurationId should be buildConfigId

Reply:
good catch!